README.md

```diff
@@ -13,22 +13,24 @@ This section guides you in using the Bold BI dashboard in your ASP.NET core samp
 
 The samples require the following requirements to run.
 
- * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
  * [.NET Core 6.0](https://dotnet.microsoft.com/download/dotnet-core)
+ * [Visual Studio Code](https://code.visualstudio.com/download)
 
  ## Using the ASP.NET Core sample
  
- * Open the solution file `BoldBI.Embed.Sample.sln` in Visual studio. 
+ * Open the ASP.NET Core sample in Visual Studio Code.
 
- * Open the EmbedProperties.cs file in the following location, /Models/EmbedProperties.cs.
-
- * Please change the following properties in the `EmbedProperties.cs` file as per your Bold BI Server.
+ * This [link](https://help.boldbi.com/cloud-bi/site-administration/embed-settings/) using able to download the embedConfig.json file. 
+ 
+ * Copy the downloaded embedConfig.json file, and paste it into the sample [application](https://github.com/boldbi/aspnet-core-sample/tree/master/BoldBI.Embed.Sample/BoldBI.Embed.Sample).
```

Comment:
Copy the downloaded embedConfig.json file, and place it into the following **location** of the application.

Link set to **Location**.

Include image for location in FileExplorer.

Reply:
Content properly updated. Image also attached.